search/ranker.cpp

```diff
@@ -540,7 +540,11 @@ void Ranker::UpdateResults(bool lastUpdate)
     }
 
     if (!lastUpdate)
+    {
       BailIfCancelled();
+      if (count % m_params.m_batchSize == 0)
```

Comment:
// For categorial requests, it is usual that several batches arrive
// in the first call to UpdateResults(). Emit them as soon as they are available
// to improve responsiveness.